zcash_history/Cargo.toml

```diff
@@ -19,6 +19,9 @@ byteorder = "1"
 blake2 = { package = "blake2b_simd", version = "1" }
 proptest = { version = "1.0.0", optional = true }
 
+# pin the winnow dependency due to MSRV of 1.64 in 0.4.2
+winnow = { version = "=0.4.1" }
+
```

Comment:
This commit needs to be removed (it now conflicts with #833), probably meaning that this PR should be rebased.